dashboard/config/levels/custom/pythonlab/Allthethings Python Lab 1.level

```diff
@@ -27,8 +28,12 @@
         }
       }
     },
-    "long_instructions": "## Welcome to Python Lab\r\n\r\nPress the Run button and see what happens."
+    "long_instructions": "## Welcome to Python Lab\r\n\r\nPress the Run button and see what happens.",
+    "reference_links": [
+      "/courses/csa-2023/guides/instantiating-objects"
+    ],
+    "map_reference": "/docs/concepts/html"
```

Comment:
Nit: After reseeding this level, the map link wasn't working for me. Adding a slash fixes that.

```suggestion
    "map_reference": "/docs/concepts/html/"
```